### PR TITLE
[ci] Add rule to not allow code after `then` in conditional

### PR DESCRIPTION
**Pull request by claywar**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Adds CI rule to check for any non-comment text after the word `then` in lua scripts and corrects all issues raised from it.

## Steps to test these changes
No changes should be observed

### CODE REVIEW

File: scripts/actions/mobskills/astral_flow_pet.lua [hunk 1]
```diff
@@ -42,18 +55,12 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
...
-    elseif petFamily == 38 or petFamily == 383 then skillId = 913 -- ifrit     inferno
-    elseif petFamily == 40 or petFamily == 384 then skillId = 915 -- leviathan tidal wave
-    elseif petFamily == 43 or petFamily == 386 then skillId = 918 -- ramuh     judgment bolt
-    elseif petFamily == 44 or petFamily == 387 then skillId = 917 -- shiva     diamond dust
-    elseif petFamily == 45 or petFamily == 388 then skillId = 914 -- titan     earthen fury
-    else
-        printf("[astral_flow_pet] received unexpected pet family %i. Defaulted skill to Searing Light.", petFamily)
+    for mobSkillId, petFamilyList in ipairs(petAstralFlowAbility) do
+        if utils.contains(petFamily, petFamilyList) then
+            skillId = mobSkillId
```
Comment (Xaver-DaRed) [inline scripts/actions/mobskills/astral_flow_pet.lua:62]: no break for the loop when a match happens?

Comment (Xaver-DaRed) [reply scripts/actions/mobskills/astral_flow_pet.lua:62]: Also, `skillId` default value is already 919. Seems reduntant to set it to 919 again if a match happens.

**Review by Xaver-DaRed — APPROVED** [review]
Nitpicks apart, lgtm

File: scripts/actions/spells/black/bio_iii.lua [hunk 2]
```diff
@@ -45,20 +45,31 @@ spellObject.onSpellCast = function(caster, target, spell)
     -- http://wiki.ffo.jp/html/1954.html
     -- this is a tiered calculation that has at least three tiers,
     -- so I'll use breakpoints for human readability
-    local dotdmg = 0
-    if     skillLvl > 400 then dotdmg = 17
```
Comment (TeoTwawki) [inline scripts/actions/spells/black/bio_iii.lua:49]: ya know, I even called out this exact styling in the style guide years ago. _Reeeeeeee no padding between the if and the condition_

Ty for fixing.

**Review by TeoTwawki — APPROVED** [review]

File: scripts/actions/mobskills/astral_flow_pet.lua (hunk 1 above)
Comment (claywar) [reply scripts/actions/mobskills/astral_flow_pet.lua:62]: You're right, this is a carryover from a way it was previously handled

**Pull request merged**